README.md

```diff
@@ -2,23 +2,32 @@
 
 ## Developers note
 
-### install dependancies
+### Install dependancies
 
 `yarn install` this will install depedancies including the workspace
 
-### run workspace
+### Run workspace
 
 `yarn start:{workspace_name}`
 
-### create workspace
+### Create workspace
 
 Add new folder under ./app, follow instruction from https://classic.yarnpkg.com/lang/en/docs/workspaces/
 Remember to put new start command of the workspace to the root package.json file
 
-### lint
+### Code quality and formatting
+
+Run [ESLint](https://eslint.org/) to analyze the code and catch bugs:
 
 `yarn lint`
 
-### lint-fix with formatting
+Run [ESLint](https://eslint.org/docs/latest/user-guide/command-line-interface#fixing-problems) with --fix
+
+**Please do this before making a new pr**
+
+`yarn lint-fix`
+
+### Useful links
 
-`yarn lint-fix` please do this before making a new pr
+- [Hardhat Basic Env](https://hardhat.org/hardhat-runner/docs/guides/typescript)
+- [Hardhat Toolbox](https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-toolbox)
```

Comment:
It would be nice if we also document the environment variable in Readme file as well, ex: a table with variable name and use

Reply:
but please ignore the comment if that's too much work for now 😅

Reply:
Agree, do you think we should put that in the Readme or should we create another Readme in the contract folder

Reply:
I think put in the README is fine~~